detector:
  ros__parameters:
    yolo_model: 
      weight_file: "/home/nvidia/yolov7_ros2_ws/src/tracking_ros2/config/weights/ankle_band_detection.pt"
      class_file: "/home/nvidia/yolov7_ros2_ws/src/tracking_ros2/config/classes.txt"
      conf_thresh: 0.5
      iou_thresh: 0.45
      device: ""
      img_size: 640
      trace: False
    img_sub_topic: "/image_raw"
    img_pub_topic: "/yolo_detection/detector/image"
    bbox_pub_topic: "/yolo_detection/detector/bounding_boxes"
